integration/testutil.go

```diff
@@ -320,9 +332,12 @@ func encodeImageInfoNerdctl(ii ...imageInfo) [][]string {
 
 func copyImage(sh *shell.Shell, src, dst imageInfo) {
 	opts := encodeImageInfoNerdctl(src, dst)
-	sh.
-		X(append([]string{"nerdctl", "pull", "-q", "--platform", platforms.Format(src.platform)}, opts[0]...)...).
-		X("ctr", "i", "tag", src.ref, dst.ref).
+	if src.mirror {
+		sh.X(append([]string{"nerdctl", "pull", "-q", "--platform", platforms.Format(src.platform)}, opts[0]...)...)
+	} else {
+		sh.X(append([]string{"ctr", "i", "import", "--platform", platforms.Format(src.platform)}, fmt.Sprintf("testImages/%s.tar", src.imgName))...)
+	}
```

Comment:
can we have an estimate how many times `"nerdctl", "pull", "-q"` and `"ctr", "i", "import"` are called respectively? (e.g., you can run integration test and print `nerdctl-pull-count` (or `ctr-i-import-count`) whenever they're called and check the output.

Reply:
Last time I checked nerdctl was called roughly ~50 times or so. I'll run it again and check

Reply:
Just ran the integration suite locally here and noted the counts along with the time it took for each operation (pull + import). Here are the results:

pull count: 45
import count: 83
avg pull time: 776.2 ms
avg import time: [4305.6](tel:43058674698) ms

Reply:
Looks like imports generally take longer than most pulls , which is very interesting to say the least.

Reply:
Following up on our meeting regarding this PR. I don't think those comparison statistics convey what they seem to. The scenarios where pull and import are happening could be wildly different, e.g. one might be all small images and the other all large.

Suggestion: Disable this PR and just include the counts and timing collection. Run the integration test suite.
Then compare those results to the results you have here.